src/Entity/Projection/ReferentManagedUser.php

```diff
@@ -207,10 +206,6 @@ public function getType(): string
 
     public function getStyleType(): string
     {
-        if (self::TYPE_NEWSLETTER === $this->type) {
-            return 'newsletter';
-        }
-
```

Comment:
Not related to you change, but since you edit those lines, could you use constants here, so add one for `host`?

Reply:
Done.